containers/src/Kokkos_DynamicView.hpp

```diff
@@ -10,7 +10,13 @@
 
 #include <cstdio>
 
+#include <Kokkos_Macros.hpp>
+#ifdef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULES
+import kokkos.core;
+import kokkos.core_impl;
+#else
 #include <Kokkos_Core.hpp>
+#endif
 #include <impl/Kokkos_Error.hpp>
 
```

Comment:
`uintptr_t` is also used here
```suggestion
#include <impl/Kokkos_Error.hpp>

#include <cstdint>

```